dbbackup/management/commands/mediabackup.py

```diff
@@ -95,6 +90,11 @@ def get_source_dir(self):
 
     def create_backup_file(self, source_dir, backup_basename, **kwargs):
         temp_dir = tempfile.mkdtemp(dir=dbbackup_settings.TMP_DIR)
+        # Amazon S3 data model is a flat structure. However, Amazon Console
```

Comment:
AWS has a flat structure but `/` is usable. I don't think this trick is great, especially give a extra behavior to AWS's storage in this file.

Could you remove it with a ``git commit --amend`` for avoid multiple commits ?